content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -252,13 +267,13 @@ commit.
 
 ```dvc
 $ dvc metrics diff
-Path         Metric    HEAD     workspace  Change
-scores.json  avg_prec  0.52048  0.55259    0.03211
-scores.json  roc_auc   0.9032   0.91536    0.01216
+Path             Metric    HEAD      workspace    Change
+evaluation.json  avg_prec  0.89668   0.9202       0.02353
+evaluation.json  roc_auc   0.92729   0.94096      0.01368
 ```
 
-And finally, we can compare both `precision recall` and `roc` curves with a
-single command!
+And finally, we can compare all plots with a single command (we show only
+`precision recall` and `roc` for simplicity):
```

Comment:
This sounds suspect 😄 . Is there another reason we aren't showing the other plots? I think the importance one is the most interesting of all since it shows how bigrams captures terms like "data frame" which are more discriminative than the individual words.

Reply:
Yep, agreed. I can put the importance one side by side. Was trying to keep it reasonably small  ... but it looks like we go into actual data science with your efforts on the flexible plots :)

Reply:
done